interface/billing/era_payments.php

```diff
@@ -233,107 +237,149 @@ function OnloadAction()
         margin-left:5px !Important;
     }
     </style>
+    <?php
+    //to determine and set the form to open in the desired state - expanded or centered
+    // any selection the user makes will become the user-specific default for that page
+    
+    $current_filename = 'era_payments';
+         
+    if (getUserSetting($current_filename) > -1) {
+        $user_value = getUserSetting($current_filename);
+    } elseif ($GLOBALS['expand_form']) {
+        $user_value = $GLOBALS['expand_form'];
+    } else {
+        $user_value = 0;
+    }
+    
+    //the linked files that would need to open in the same state, expanded or centered, as the current page
+    // if no linked files comment out next 4 lines
+    $filenames = array("new_payment", "search_payments", "era_payments");
+    foreach ($filenames as $filename) {
+        setUserSetting($filename, $user_value);
+    }
+    //$user_value = getUserSetting('era_payments');
+    $current_state = $user_value;
+    if ($current_state == 1) {
```

Comment:
since just using true/false in above function, just make this:
```
if ($current_state) {
```

Reply:
Done